docs/api/migration-guides/qiskit-algorithms-module.mdx

```diff
@@ -820,7 +819,7 @@ iae = IterativeAmplitudeEstimation(
 
 For complete code examples, see the following updated tutorials:
 
-- [Amplitude Estimation](https://qiskit.org/documentation/finance/tutorials/00_amplitude_estimation.html)
+- [Amplitude Estimation](https://qiskit.org/ecosystem/finance/tutorials/00_amplitude_estimation.html)
```

Comment:
This site will eventually go away in favor of a GitHub page, but it's not yet ready.